src/NSwag.CodeGeneration.CSharp/Templates/Controller.liquid

```diff
@@ -1,4 +1,8 @@
-﻿{% if GeneratePartialControllers -%}
+﻿{% if IsAspNetCore -%}
+using FromHeaderAttribute = {{ AspNetNamespace }}.FromHeaderAttribute;
```

Comment:
This won't work as the Controller.liquid template is repeated multiple times depending on the client generation mode...